src/main/java/edu/unc/lib/boxc/migration/cdm/status/GroupMappingStatusService.java

```diff
@@ -63,7 +63,7 @@ public void reportStats(int totalObjects, Verbosity verbosity) {
             try {
                 modified = Files.getLastModifiedTime(mappingsPath).toInstant();
             } catch (IOException e) {
-                outputLogger.info("Failed to check mappings: {}", e.getMessage());
+                outputLogger.info("Failed to check mappings: {}", e);
```

Comment:
Might make sense to add in a regular logger line here and just send the message to the outputLogger (and the other locations in this file)